### PR TITLE
Gem proof

### PR DESCRIPTION
**Pull request by Sammers21**
A proof that gem repository works.

Related to #154.

This PR is draft until #373 is not merged

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8,  can you review?

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I see that you unassigned this issue; the order is still assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

File: examples/gem/run.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Start artipie.
+docker run --rm -d --name artipie -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Push a gem into artipie.
+cd sample-project
+gem build sample-project.gemspec
+GEM_HOST_API_KEY=1 gem push sample-project-1.0.0.gem --host http://localhost:8080/my-gem
```
Comment (g4s8) [inline examples/gem/run.sh:13]: @Sammers21 can we export this variable to the script's scope to avoid duplication?

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Pull request merged**